README.md

```diff
@@ -41,10 +41,6 @@ Check also [presentations and videos](ur_robot_driver/doc/resources/README.md) a
          <img src="https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/actions/workflows/foxy-semi-binary-build.yml/badge.svg?event=schedule"
               alt="Foxy Semi-Binary Build"/>
       </a> <br />
```

Comment:
```Suggestion
      </a>
```

I don't think these breaks belong here anymore (for the other entries as well)

Reply:
Fixed in 41594d3